README.md

```diff
@@ -76,6 +77,29 @@ providing an API to convert user-entered number and date strings - in their
 own format - into actual numbers and dates, and conversely, to format numbers
 and dates into that string format.
 
+<a name="why_globalize"></a>
+### Why Globalize?
+
+Globalize has initially been created to fullfil jQuery UI (datepicker and
```

Comment:
Well, initially Microsoft wanted to provide a JS globalization library, it had nothing to do with jQuery UI. If we keep this sentence, the grammar needs to be improved, e.g., "has ben created to fullfil the requirements of jQuery UI" or "to provide globalization support for jQuery UI"